air/src/trace/main_trace.rs

```diff
@@ -92,19 +92,9 @@ impl MainTrace {
         self.addr(i) != self.addr(i + 1)
     }
 
-    /// First decoder helper register at row i.
-    pub fn helper_0(&self, i: usize) -> Felt {
-        self.columns.get_column(DECODER_TRACE_OFFSET + USER_OP_HELPERS_OFFSET)[i]
-    }
-
-    /// Second decoder helper register at row i.
-    pub fn helper_1(&self, i: usize) -> Felt {
-        self.columns.get_column(DECODER_TRACE_OFFSET + USER_OP_HELPERS_OFFSET + 1)[i]
-    }
-
-    /// Third decoder helper register at row i.
-    pub fn helper_2(&self, i: usize) -> Felt {
-        self.columns.get_column(DECODER_TRACE_OFFSET + USER_OP_HELPERS_OFFSET + 2)[i]
+    /// The i-th decoder helper register at `row`.
+    pub fn helper_register(&self, i: usize, row: usize) -> Felt {
+        self.columns.get_column(DECODER_TRACE_OFFSET + USER_OP_HELPERS_OFFSET + i)[row]
```

Comment:
A thought for the future: we should probably introduce a newtype for to represent row. Maybe we call it `Cycle` or `Step`. It could look something like this:
```
pub struct Step(u32);
```

And then we can implement both `From<Step> for u32` and `From<Step> for usize`.

Let's create an issue for this.